src/CommunityToolkit.Maui.UnitTests/Converters/IsStringNotNullOrEmptyConverter_Tests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Globalization;
+using CommunityToolkit.Maui.Converters;
+using Xunit;
+
+namespace CommunityToolkit.Maui.UnitTests.Converters;
+
+public class IsStringNotNullOrEmptyConverter_Tests : BaseTest
+{
+	[Theory]
+	[InlineData("Test", true)]
+	[InlineData(null, false)]
+	[InlineData("", false)]
+	[InlineData(" ", true)]
+	public void IsNotNullOrEmptyConverter_ValidStringValue(string? value, bool expectedResult)
+	{
+		var isNotNullOrEmptyConverter = new IsStringNotNullOrEmptyConverter();
+
+		var result = (bool)isNotNullOrEmptyConverter.Convert(value, null, null, null);
+
+		Assert.Equal(expectedResult, result);
```

Comment:
This wasn't your original test but I am beginning to love FluentAssertions

```suggestion
        result.Should().Be(expectedResult);
```

Actually in this scenario I am not sure it is more readable. What are your thoughts?

Reply:
Yea, it's probably a toss-up for readability. 

As a tie breaker, I'll leave it to use the classic `Assert.Equals` since the other unit test also uses `Assert.*`.